Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.16
+RUN apk upgrade --no-cache
```

Comment:
Use one RUN instead of multiple: https://github.com/hadolint/hadolint/wiki/DL3059